katdal/h5datav3.py

```diff
@@ -112,6 +112,10 @@ def dummy_dataset(name, shape, dtype, value):
     return dummy_file.create_dataset(name, shape=shape, maxshape=shape,
                                      dtype=dtype, fillvalue=value, compression='gzip')
 
+
+class AttributeFound(Exception):
```

Comment:
I would put an underscore in front of the name: this is used internally and is not part of the API.

Reply:
Good point.